src/static/javascripts/components/questions.jsx

```diff
@@ -80,6 +91,9 @@ var MultipleChoice = React.createClass({
 *
 *
 * Single Choice
+* This is the same as multiple choice, but the handlers are different
+* so that once a new option is chosen all other options are set to false
+* since it simulates a radio form
 */
 var SingleChoice = React.createClass({
     
```

Comment:
Fixed the radio card, set the name to be the unique Survey ID, hopefully that's a permanent fix but we should consider edge cases.